src/Compilers/Core/Portable/PEWriter/MetadataWriter.PortablePdb.cs

```diff
@@ -990,5 +991,28 @@ private void EmbedMetadataReferenceInformation(CommonPEModuleBuilder module)
                 kind: _debugMetadataOpt.GetOrAddGuid(PortableCustomDebugInfoKinds.CompilationMetadataReferences),
                 value: _debugMetadataOpt.GetOrAddBlob(builder));
         }
+
+        private void TypeDocumentInformation(CommonPEModuleBuilder module)
```

Comment:
>TypeDocumentInformation [](http://example.com/codeflow?start=21&length=23)

The name `TypeDocumentInformation` is missing verb and does not convey what the method is doing with the type document information.